### ISSUE TITLE
Rate limits?

### ISSUE THREAD

**Issue by zeke** [zeit/hazel#12]
It appears that hazel is making unauthenticated requests to the GitHub API.. is that correct?

If so, a running instance will quickly go over its unauthenticated request limit.

**Comment by leo** [comment]
Check the `INTERVAL` env variable [here](https://github.com/zeit/hazel#options).

**Comment by zeke** [comment]
So.. unauthenticated requests then, right?

**Comment by leo** [comment]
@zeke Correct. All unauthenticated. That's what makes Hazel so fast and powerful: It only abstracts the necessary. If you need server-side caching / private repos, there's Nuts... 😊

**Comment by zeke** [comment]
I see. So hazel will only make a request to the GitHub API every `INTERVAL` minute(s), even though apps will be asking hazel for that info much more frequently.

Does every Now app get its own IP address?

**Comment by leo** [comment]
> I see. So hazel will only make a request to the GitHub API every `INTERVAL` minute(s), even though apps will be asking hazel for that info much more frequently.

Correct.

> Does every Now app get its own IP address?

Depends. Sometimes they're on the same IP.

**Comment by zeke** [comment]
I ask because unauthenticated GitHub API requests are tallied by IP. It might be worth defaulting to a longer interval to reduce the likelihood of contention across apps.
